launchpad/reward_calculation.gno

```diff
@@ -58,28 +66,43 @@ var rewardStates = RewardStates{
 	states: avl.NewTree(),
 }
 
-func (states RewardStates) Get(projectId string, tierStr string) *RewardState {
+func (states RewardStates) Get(projectId string, tierStr string) (*RewardState, error) {
 	key := projectId + ":" + tierStr
 	statesI, exists := states.states.Get(key)
 	if !exists {
-		return nil
+		return nil, ufmt.Errorf("reward state not found for projectId %s and tierStr %s", projectId, tierStr)
 	}
-	return statesI.(*RewardState)
+	return statesI.(*RewardState), nil
 }
 
+/// XXXX
```

Comment:
what is this

Reply:
there was an issue the `rewardState` wasn't calculated because the `Set` does not called before.
https://github.com/gnoswap-labs/gnoswap/pull/451/commits/6de8ce8534fd7589331da2a56c25f8a61c2ef08b